azure-pipelines.perf-test.yml

```diff
@@ -22,15 +22,19 @@ jobs:
 
       - script: |
           NorthstarAffected=$(yarn --silent check-if-v0-v8-v9-affected --@fluentui/react-northstar)
-          V8Affected=$(yarn --silent check-if-v0-v8-v9-affected --@fluentui/react --@fluentui/react-components)
+          V8Affected=$(yarn --silent check-if-v0-v8-v9-affected --@fluentui/react)
```

Comment:
this is obsolete right (`yarn --silent check-if-v0-v8-v9-affected`) ? we use `node` directly to run the script

Reply:
I initially used `node` to run the script in the pipeline since using `yarn` created a lot of extra churn. Ling gave me the secret sauce of using `--silent` to get rid of that churn so I went back to using a script alias here 😄. Although I'm fine just getting rid of the alias and just using `node` here to run the script too if you think that's better